<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>463</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>463</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>463</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cotações</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/newPrefix/3592869-compose-create-edit-edit-file-office-pencil-writing-creative_107746.png</normaloff>:/newPrefix/3592869-compose-create-edit-edit-file-office-pencil-writing-creative_107746.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_editar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>429</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dpf">
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>429</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DPF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_editar_2">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>429</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
